geti_sdk/data_models/shapes.py

```diff
@@ -39,15 +39,15 @@
 coordinate_converter = round_to_n_digits(N_DIGITS_TO_ROUND_TO)
 
 
-@attr.s(auto_attribs=True)
+@attr.define(auto_attribs=True, slots=False)
```

Comment:
Why does this class need `slots=False`?

Reply:
The shapes that inherit from this class are setting attributes, should I add `slots=False` to all shapes?